### PR TITLE
Mitra reload

### PR DESCRIPTION
**Pull request by mitra42**
**Description**
Adds a reload button for DwebMirror , integrates it into DwebNav, when clicked it passes on a reload flag that gets passed through to transport layer and adds appropriate headers to disable the DwebMirror cache and ensure its reloaded.  

DEPENDS ON #188 

**Technical**
Some refactoring of AnchorDetails was required to add some missing functionality to pass on the optional "reload" flag. 

**Testing**
Live in dweb-archive when included as part of the mitra--release branch 

**Evidence**
This is live in the the current dweb-mirror release

### CODE REVIEW

**Comment by iisa** [comment]
hi @mitra, there's a conflict when i merged in Navwrap.
Also, your commit history should only reflect the ones that affected the files in this commit.   If it isn't so, we may continue to encounter conflicts.

**Comment by mitra42** [comment]
I hadn't noticed, but this PR depends on #188

**Comment by mitra42** [comment]
In general - the commit history is going to depend on all the PR's that have gone before, but not yet been merged. That's because each PR is building on what has been done already, on most teams I've not seen this as a big problem because commits are merged reasonably fast, but obviously with a typical 2-3 week delay on IAUX there is a lot of work happening in the meantime, and that can't be built on the base "master" branch because its bug fixes and improvements to code already submitted for PR but not yet accepted.

**Comment by iisa** [comment]
what happens with a rebase of master into this branch? @mitra42

**Comment by mitra42** [comment]
First ... I don't believe there is any such thing as a Rebase of a branch that has already been shared. When I try and do that it throws up a big warning, and if I ignore it I've caused massive issues with Git. 

If I Merge Master into this branch, its what causes the extra lines in the commit history that you are (reasonably) complaining about.   I don't look at the commit history, because of that, I look at the "Files Changed" branch, which shows all the changes + all the changes that it depends on, that are in PR's not yet merged. Those files disappear from "Files Changed" as soon as the other PRs are merged.

File: packages/ia-components/sandbox/AnchorDetails.jsx [hunk 1]
```diff
@@ -15,29 +35,32 @@ export default class AnchorDetails extends IAReactComponent {
     */
     constructor(props)
     {
-        super(props);
+        super(props); // { identifier, reload }
+        this.setState({
+            urlProps: ObjectFilter(this.props, (k,v) => AnchorDetails.urlparms.includes(k)),
```
Comment (shaneriley) [inline packages/ia-components/sandbox/AnchorDetails.jsx]: `urlparms` could be renamed `urlparams` to be more understandable.

Comment (mitra42) [reply packages/ia-components/sandbox/AnchorDetails.jsx]: Rejected: non functional suggestion and personal opinion on how to abbreviate "parameters"

**Pull request closed without merging**